Makefile

```diff
@@ -1,12 +1,12 @@
 pkg_name := "auter"
 
 # Version info:
-git_tag := $(shell git describe --exact-match --tags 2>/dev/null | sed "s/^v\?//g")
+#git_tag := $(shell git describe --exact-match --tags 2>/dev/null | sed "s/^v\?//g")
+git_tag := $(shell awk '/Version:/ {print $$2}' auter.spec)
 git_commit := $(shell git log --pretty=format:'%h' -n 1)
-release := "1"
+release := $(shell awk '/Release:/ {gsub(/%.*/,""); print $$2}' auter.spec)
```

Comment:
Accidental $$ after the print in the awk script.

Reply:
This is a workaround for escaping the $ in the Makefile